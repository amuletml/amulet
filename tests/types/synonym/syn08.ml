type foo 'a 'b <- 'a * 'b

class x 'a begin end
instance x (foo int) begin end
