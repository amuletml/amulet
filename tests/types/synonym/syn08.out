syn08.ml[4:1 ..4:30]: error
  │ 
4 │ instance x (foo int) begin end
  │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  Type synonym foo appears with one argument,
  but in its definition it has 2
